src/lib/Containers/Conversation.tsx

```diff
@@ -144,6 +146,21 @@ export class Conversation extends React.Component<Props, State> {
     })
     const lastMessage = messages[messages.length - 1]
 
+    // Better way of marking a message as read?
+    if (conversation.is_last_message_to_user && !conversation.last_message_open && !this.state.markedMessageAsRead) {
```

Comment:
This is located within the `render` method. Basically, we mark the message as read right away (assuming it's to the user, and is unread). We use state to just make sure we fire this once.

There's probably some sort of more logical way to put this, but rollin' this for now #wip #testing

Reply:
Right, not sure we would need to have this check in render as it calls pretty often in the Conversation lifecycle, I think a good place for this would be the `componentDidMount` method which only gets called once when the view is loaded

Reply:
👍